youtube_dl_gui/locale/zh_CN/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,582 @@
+msgstr "播放"
+
+#: mainframe.py:109
+msgid "Up"
+msgstr "提前"
+
+#: mainframe.py:110
+msgid "Down"
+msgstr "下降"
+
+#: mainframe.py:111
+msgid "Reload"
+msgstr "重新加载"
+
+#: mainframe.py:112 mainframe.py:443 mainframe.py:644
+msgid "Pause"
+msgstr "暂停"
+
+#: mainframe.py:113 mainframe.py:860 mainframe.py:861 optionsframe.py:585
+msgid "Start"
+msgstr "开始"
+
+#: mainframe.py:114
+msgid "About"
+msgstr "关于"
+
+#: mainframe.py:115
+msgid "View Log"
+msgstr "查看日志"
+
+#: mainframe.py:117
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "成功下载{0}个链接 用时{1}天{2}小时{3}分钟{4}秒"
+
+#: mainframe.py:119
+msgid "Downloads completed"
+msgstr "下载完成"
+
+#: mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "总进度：{0:.1f}% | 队列 ({1}) 暂停 ({2}) 活动 ({3}) 完成 ({4}) 错误 ({5})"
+
+#: mainframe.py:121
+msgid "Stopping downloads"
+msgstr "正在停止下载"
+
+#: mainframe.py:122
+msgid "Downloads stopped"
+msgstr "下载已经停止"
+
+#: mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "请至少提供一个链接"
+
+#: mainframe.py:124
+msgid "Downloads started"
+msgstr "下载已经开始"
+
+#: mainframe.py:125
+msgid "Choose Directory"
+msgstr "选择目录"
+
+#: mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr "正在下载中 请等待所有下载完毕"
+
+#: mainframe.py:128
+msgid "Update already in progress"
+msgstr "更新进行中"
+
+#: mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "正在下载最新版本的youtube-dl 请稍候"
+
+#: mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "YouTube-DL下载失败 [{0}]"
+
+#: mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "YouTube-DL下载成功"
+
+#: mainframe.py:134
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr "无法打开目录 '{dir}' 此路经不存在"
+
+#: mainframe.py:136
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr "关闭系统时发生错误 请确保密码正确"
+
+#: mainframe.py:138
+msgid "Shutting down system"
+msgstr "正在关闭系统"
+
+#: mainframe.py:141
+msgid "Extension"
+msgstr "扩展名"
+
+#: mainframe.py:142
+msgid "Size"
+msgstr "大小"
+
+#: mainframe.py:143
+msgid "Percent"
+msgstr "百分比"
+
+#: mainframe.py:144
+msgid "ETA"
+msgstr "剩余时间"
+
+#: mainframe.py:145
+msgid "Speed"
+msgstr "速度"
+
+#: mainframe.py:146
+msgid "Status"
+msgstr "状态"
+
+#: mainframe.py:235
+msgid "Get URL"
+msgstr "获取地址"
+
+#: mainframe.py:236
+msgid "Get command"
+msgstr "获取命令"
+
+#: mainframe.py:237
+msgid "Open destination"
+msgstr "打开输出目录"
+
+#: mainframe.py:238
+msgid "Re-enter"
+msgstr "重新输入"
+
+#: mainframe.py:453
+msgid "Resume"
+msgstr "继续"
+
+#: mainframe.py:475
+msgid "Video"
+msgstr "视频"
+
+#: mainframe.py:479
+msgid "Audio"
+msgstr "音频"
+
+#: mainframe.py:511
+msgid "No items selected. Please pick an action"
+msgstr "没有选择项目 请选择一个操作"
+
+#: mainframe.py:511
+msgid "Remove all"
+msgstr "移除全部"
+
+#: mainframe.py:511
+msgid "Remove completed"
+msgstr "移除已完成"
```

Comment:
Google translates this to "The removal is complete" which does not match with the *msgid* string.
Are your sure this is correct?